src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <div class="calendar-field calendar-field--start-day_day calendar-field--month-length--day">
+        <div class="calendar-cell"></div>
```

Comment:
```suggestion
        <div class="calendar__cell"></div>
```
if it's an element of calendar block